src/libaktualizr/bootloader/bootloader.cc

```diff
@@ -157,5 +160,16 @@ void Bootloader::rebootFlagClear() {
   // clear in storage + volatile flag
 
   storage_.clearNeedReboot();
+  // What's the point to remove the sentinel file here if rebootFlagClear is called
```

Comment:
It's an API, so `rebootFlagClear()` could in theory be called just after `rebootFlagSet()`. It could be expressed in term of invariants.

Reply:
OK. Will remove the comment.